style.css

```diff
@@ -0,0 +1,495 @@
+/* Global Style */
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap');
+
+:root {
+  --primary-font: "Lato", sans-serif;
+  --heading-font: "COCOGOOSE", sans-serif;
+  --primary-color: #ec5242;
+  --secondary-color: #272a31;
+  --bg-color: #d3d3d3;
+  --font-color: #999;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: var(--primary-font);
+  color: #000;
+  font-size: 20px;
+  line-height: 1.5;
+}
+
```

Comment:
Kindly use the cocogoose and lato font family

Reply:
I already used both the Lato and COCOGOOSE font family